.gitlab/macrobenchmarks.yml

```
@@ -24,7 +24,7 @@ variables:
   # If you have a problem with Gitlab cache, see Troubleshooting section in Benchmarking Platform docs
   image: $MACROBENCHMARKS_CI_IMAGE
   script: |
-    git clone --branch python/macrobenchmarks https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.ddbuild.io/DataDog/benchmarking-platform platform && cd platform
+    git clone --branch igoragoli/python-macrobenchmarks-add-warmup https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.ddbuild.io/DataDog/benchmarking-platform platform && cd platform
```

Once the other PR is merged, the branch for bp should be reverted, no?

```suggestion
    git clone --branch python/macrobenchmarks https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.ddbuild.io/DataDog/benchmarking-platform platform && cd platform
```

---

Yep, thanks!